tests/features/menus_and_blocks/menus.feature

```diff
@@ -1,45 +1,29 @@
+      | footer-menu-left   | BEHAT LOCATIONS | base:behat-locations                              | 1        |
+      | footer-menu-right  | BEHAT JOIN      | base:behat-join                                   | 1        |
+      | footer-menu-center | BEHAT CAREERS   | base:behat-careers                                | 1        |
+    And I create menu_link_content:
+      | menu_name          | title           | uri                                               | expanded | parent_uri                          | parent_title    | icon_image    |
+      | main               | BEHAT CHILDCARE | base:behat-programs/behat-childcare               | 1        | base:behat-programs                 | BEHAT PROGRAMS  | childcare.png |
+      | main               | BEHAT DAYCARE   | base:behat-programs/behat-childcare/behat-daycare | 1        | base:behat-programs/behat-childcare | BEHAT CHILDCARE |               |
+    Then I am an anonymous user
 
-  Scenario: Create menu item for second level
-    Given I am logged in as a user with the "Editor" role
-    And I create an item "Programs" in the "main" menu
-    And I go to "/admin/structure/menu/manage/main/add"
-    And I fill in the following:
-      | Menu link title | Childcare   |
-      | Link            | /childcare  |
-    And I select "-- Programs" from "Parent link"
-    And I attach the file "childcare.png" to "Icon image"
-    And I press "Save"
-    When I am an anonymous user
-    And I go to the homepage
-    Then I should see "Childcare" in the "dropdown_menu"
-    
-  Scenario: Create footer menu left item and check
-    Given I am logged in as a user with the "Editor" role
-    And I create an item "LOCATIONS" in the "footer-menu-left" menu
-    When I am an anonymous user
-    And I go to homepage
-    Then I should see "LOCATIONS" in the "footer"
-
-  Scenario: Create footer menu right item and check
-    Given I am logged in as a user with the "Editor" role
-    And I create an item "JOIN" in the "footer-menu-right" menu
-    When I am an anonymous user
-    And I go to homepage
-    Then I should see "JOIN" in the "footer"
-
-  Scenario: Create footer menu center item and check
-    Given I am logged in as a user with the "Editor" role
-    And I create an item "CAREERS" in the "footer-menu-center" menu
-    When I am an anonymous user
-    And I go to homepage
-    Then I should see "CAREERS" in the "footer"
\ No newline at end of file
+  Scenario: Check menu items are in the appropriate place
+    Given I am on the homepage
+    Then I should see "BEHAT GIVE" in the "header"
+    And I should see "BEHAT PROGRAMS" in the "header"
+    And I should see an "ul.nav.dropdown-menu .menu-item-behat-childcare" element
+    And Element "ul.nav.dropdown-menu .menu-item-behat-childcare" has text "BEHAT CHILDCARE"
+    And I should see an "ul.nav.dropdown-menu .menu-item-behat-childcare .section-icon" element
```

Comment:
This is "always true". No matter if we have image or not.
![image](https://cloud.githubusercontent.com/assets/4558228/24818787/862e2c6c-1bea-11e7-8d5b-ca84c25d32a4.png)
So what we can do to test this:
1) Run this scenarios through Selenium. And evaluate some custom js that checking if image is there.
2) Make a screenshot of this element "ul.nav.dropdown-menu .menu-item-behat-childcare" . And compare with expected one.